core/test/unit/web/middleware/image/normalize_spec.js

```diff
@@ -68,14 +68,14 @@ describe('normalize', function () {
         });
     });
 
-    ['.gif', '.svg', '.svgz'].forEach(function (extension) {
-        it(`should skip processing when file extension is ${extension}`, function (done) {
+    it(`should skip processing when file extension is ${extension}`, function (done) {
```

Comment:
The same problem with `undefined` here as in other cases ;)

Reply:
rewrote this, will resubmit for you to review